src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -326,6 +351,52 @@ private static bool GetBooleanQueryParameterValue(NameValueCollection queryStrin
             return request.CreateResponse(HttpStatusCode.Accepted);
         }
 
+        private async Task<HttpResponseMessage> HandleStartOrchestratorRequestAsync(
+            HttpRequestMessage request,
+            string functionName,
+            string instanceId)
+        {
+            try
+            {
+                DurableOrchestrationClientBase client = this.GetClient(request);
+
+                object input = null;
+                if (request.Content != null)
+                {
+                    string contentAsString = await request.Content.ReadAsStringAsync();
+                    input = JsonConvert.DeserializeObject(contentAsString);
+                }
+
+                string id = await client.StartNewAsync(functionName, instanceId, input);
+
+                TimeSpan? timeout = this.GetTimeSpan(request, "timeout");
+                TimeSpan? pollingInterval = this.GetTimeSpan(request, "pollingInterval");
+
+                if (timeout.HasValue && pollingInterval.HasValue)
+                {
+                    return await client.WaitForCompletionOrCreateCheckStatusResponseAsync(request, id, timeout.Value, pollingInterval.Value);
+                }
+                else
+                {
+                    var res = client.CreateCheckStatusResponse(request, id);
+                    res.Headers.RetryAfter = new RetryConditionHeaderValue(TimeSpan.FromSeconds(10));

```

Comment:
Good catch; thank you. I added the line because we include it in the [HttpStart sample](https://github.com/Azure/azure-functions-durable-extension/blob/fbe8bf10240b51e9abc3462a32046e38bde70135/samples/precompiled/HttpStart.cs#L30). I'll go ahead and remove it there as well.